extensions/amp-ad-network-adsense-impl/0.1/test/test-adsense-a4a-config.js

```diff
@@ -123,4 +126,35 @@ describe('adsense-a4a-config', () => {
       expect(elem.getAttribute(EXPERIMENT_ATTRIBUTE)).to.equal('2092615');
     });
   });
+
+  describe('#fastFetchDelayedRequestEnabled', () => {
+    Object.entries({
+      [ADSENSE_EXPERIMENT_FEATURE.DELAYED_REQUEST_EXTERNAL_CONTROL]: {
```

Comment:
It's not an array, it just ES6 nomenclature for specifying an inline object with a variable key name